File: WebApp/src/app/shared/models/system/sysUserNotification.ts
// tslint:disable-next-line: file-name-casing
export class SysNotification {
    id: string;
    description: string;
    title: string;
    type: string;
    action: string;
    actionLink: string;
    isClosed: boolean;
    isRead: boolean;
    datetimeCreated: string;
    datetimeModified: string;
    userCreated: string;
    userModified: string;
    userIds: string[];
}

export class SysUserNotificationModel extends SysNotification {
}

export class SysUserNotification {
    id: string;
    type: string;
    title: string;
    description: string;
    action: string;
    actionLink: string;
    status: STATUS_NOTI;
    datetimeCreated: string;
    datetimeModified: string;
    userCreated: string;
    userModified: string;
    userId: string;
}

type STATUS_NOTI = 'New' | 'Read' | 'Closed'; 

File: WebAPI/eFMS.API.SystemWeb/eFMS.API.SystemFileManagement/eFMS.API.SystemFileManagement.DAL/Models/SysUserNotification.cs
﻿using System;

namespace eFMS.API.SystemFileManagement.Service.Models
{
    public partial class SysUserNotification
    {
        public Guid Id { get; set; }
        public Guid? NotitficationId { get; set; }
        public string UserId { get; set; }
        public string Status { get; set; }
        public DateTime? DatetimeCreated { get; set; }
        public DateTime? DatetimeModified { get; set; }
        public string UserCreated { get; set; }
        public string UserModified { get; set; }
    }
}
